treeherder/config/settings.py

```diff
@@ -102,14 +102,9 @@ def server_supports_tls(url):
 
 AUTHENTICATION_BACKENDS = (
     'django.contrib.auth.backends.ModelBackend',
-    'django_browserid.auth.BrowserIDBackend',
+    'django.contrib.auth.backends.RemoteUserBackend',
```

Comment:
Yeah, agreed about the custom backend.  I had thought about that yesterday as well.  Thanks for taking a look at this while it's still pretty messy.  My comment in IRC, I somehow missed the part where you said "I'm pretty sure we need to be using a custom backend"  So my comment in IRC probably seemed like I was looney...  :)